magda-gateway/src/createBaseProxy.ts

```diff
@@ -44,5 +47,28 @@ export default function createBaseProxy(): httpProxy {
         });
     });
 
+    proxy.on("proxyReq", function(proxyReq, req, res) {
+        proxyReq.setHeader("TenantId", "undefined");
+        const host = req.headers.host;
+        let endIndex = host.lastIndexOf(":");
+        if (endIndex < 0) endIndex = host.length;
+
+        let domainName = host.substring(0, endIndex);
+
+        if (domainName == magdaAdminPortalName || domainName == "localhost") {
```

Comment:
Better to use `===` than `==` in javascript (it's a dumb language in a lot of ways): https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a :)

Reply:
done.